### PR TITLE
revies dependencies

### PR DESCRIPTION
**Pull request by fmessmer**
sorting, cleaning up and review (used roscompile)

supersedes/related to #315

Update this PR in case other PRs get merged first!

### CODE REVIEW

**Comment by floweisshardt** [comment]
LGTM

File: cob_bringup/package.xml [hunk 1]
```diff
@@ -12,72 +12,70 @@
...
   <run_depend>cob_hardware_config</run_depend>
-  <run_depend>cob_default_robot_config</run_depend>
-  <run_depend>cob_default_env_config</run_depend>
-  <run_depend>cob_phidgets</run_depend>
-  <run_depend>prace_gripper_driver</run_depend>
-  <run_depend>frida_driver</run_depend>
+  <run_depend>cob_head_axis</run_depend>
   <run_depend>cob_image_flip</run_depend>
-  <run_depend>schunk_sdhx</run_depend>
+  <run_depend>cob_lbr</run_depend>
```
Comment (NadiaHG) [inline cob_bringup/package.xml:36]: do we still need cob_lbr?

Comment (fmessmer) [inline cob_bringup/package.xml:36]: Well, there is still the launch file https://github.com/ipa320/cob_robots/blob/indigo_dev/cob_bringup/drivers/lbr_driver.launch
and we have a substitution for it in `cob_substitute...

**Comment by floweisshardt** [comment]
hard to oversee if anything is missing. I'll merge and give it a try on the buildfarm.

**Pull request merged**